### PR TITLE
python-weakrefset: init

### PR DESCRIPTION
**Pull request by Kiri2002**
dependence for python-pyexcel-ezodf

## Involved packages

 - Packages_Name

## Involved issue

Close #

## Details

- [ ] Tested in the local machine (largest 16G RAM) and it is passed without any issue
- [ ] Provide New Package
- [ ] Fix the Packages
  - [ ] PKGBUILD
  - [ ] lilac.yaml
  - [ ] lilac.py
- [ ] Would like to continue to work with us

### CODE REVIEW

File: BioArchLinux/python-weakrefset/PKGBUILD [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+             'python-installer'
+             'python-pytest-runner'
+            )
+source=("https://files.pythonhosted.org/packages/source/${_module::1}/$_module/$_module-$pkgver.tar.gz")
+sha256sums=('1199c768153c2f4ac123f2a57e93a28185d843b09864ea3f4219d383d1cc16aa')
+
+build() {
+    cd "$_module-$pkgver"
+    sed -i 's/force_manifest/force-manifest/' setup.cfg
+    python setup.py build
```
Comment (hubutui) [inline BioArchLinux/python-weakrefset/PKGBUILD:25]: You have list `python-installer` as makedepends, why not use PEP 517 as the [wiki](https://wiki.archlinux.org/title/Python_package_guidelines#Standards_based_(PEP_517)) described?

**Comment by starsareintherose** [comment]
Please follow and consider hubutui's comment

**Comment by Kiri2002** [comment]
[pypi](https://pypi.org/project/weakrefset/)  shows: License: BSD License (UNKNOWN)
the source file not contains a license, should i add a BSD License file to source ?

File: BioArchLinux/python-weakrefset/PKGBUILD (hunk 1 above)
Comment (Kiri2002) [reply BioArchLinux/python-weakrefset/PKGBUILD:23]: removed ```python-installer``` since it only can build with old way --- setup.cfg

**Comment by kbipinkumar** [comment]
> [pypi](https://pypi.org/project/weakrefset/) shows: License: BSD License (UNKNOWN) the source file not contains a license, should i add a BSD License file to source ?

Yes. if the source tarball doesnt have a license file you provide your LICENSE file in source=() array of PKGBUILD and install it in package() section

**Comment by Kiri2002** [comment]
sorry, I make a fault that the package is not the dependence, so i just close it.

**Pull request closed without merging**